### ISSUE TITLE
Elixir-LS in VSCode fails with ** (Protocol.UndefinedError) protocol JasonVendored.Encoder not implemented for #PID

### ISSUE THREAD

**Issue by philippseith** [elixir-lsp/elixir-ls#871]
### Environment

* Erlang/OTP 24 [erts-12.3.2.10] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:1] [jit]
* Elixir 1.14.1 (compiled with Erlang/OTP 24)
* ElixirLS release v0.14.3
* Ubuntu 20.04.6 LTS, x86
* VSCode 1.77.3
* Plugin: https://github.com/elixir-lsp/vscode-elixir-ls

### Current behavior

When the Language Server is started, VSCode shows a notification "ElixirLS has crashed. See Output panel.". Same happens when executing "Elixir: Restart Language Server"
After the Language Server restarted itself automatically, Intelisense is working in the editor 

Logs:
```
[Info  - 5:07:59 PM] ElixirLS will restart
[Error - 5:07:59 PM] GenServer ElixirLS.LanguageServer.Server terminating
** (Protocol.UndefinedError) protocol JasonVendored.Encoder not implemented for #PID<0.455.0> of type PID, JasonVendored.Encoder protocol must always be explicitly implemented. This protocol is implemented for the following type(s): Any, Atom, BitString, Date, DateTime, Decimal, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.Cancel, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.Cancel.LSP, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.DidChange, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.DidChange.LSP, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.DidChangeConfiguration, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.DidChangeConfiguration.LSP, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.DidChangeWatchedFiles, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.DidChangeWatchedFiles.LSP, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.DidClose, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.DidClose.LSP, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.DidOpen, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.DidOpen.LSP, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.DidSave, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.DidSave.LSP, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.Initialized, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.Initialized.LSP, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.PublishDiagnostics, ElixirLS.LanguageServer.Experimental.Protocol.Notifications.PublishDiagnostics.LSP, ElixirLS.LanguageServer.Experimental.Protocol.Proto.LspTypes.ClientInfo, ElixirLS.LanguageServer.Experimental.Protocol.Proto.LspTypes.Registration, ElixirLS.LanguageServer.Experimental.Protocol.Proto.LspTypes.ResponseError, ElixirLS.LanguageServer.Experimental.Protocol.Requests.CodeAction, ElixirLS.LanguageServer.Experimental.Protocol.Requests.CodeAction.LSP, ElixirLS.LanguageServer.Experimental.Protocol.Requests.FindReferences, ElixirLS.LanguageServer.Experimental.Protocol.Requests.FindReferences.LSP, ElixirLS.LanguageServer.Experimental.Protocol.Requests.Formatting, ElixirLS.LanguageServer.Experimental.Protocol.Requests.Formatting.LSP, ElixirLS.LanguageServer.Experimental.Protocol.Requests.GotoDefinition, ElixirLS.LanguageServer.Experimental.Protocol.Requests.GotoDefinition.LSP, ElixirLS.LanguageServer.Experimental.Protocol.Requests.Initialize, ElixirLS.LanguageServer.Experimental.Protocol.Requests.Initialize.LSP, ElixirLS.LanguageServer.Experimental.Protocol.Requests.RegisterCapability, ElixirLS.LanguageServer.Experimental.Protocol.Requests.RegisterCapability.LSP, ElixirLS.LanguageServer.Experimental.Protocol.Responses.CodeAction, ElixirLS.LanguageServer.Experimental.Protocol.Responses.FindReferences, ElixirLS.LanguageServer.Experimental.Protocol.Responses.Formatting, ElixirLS.LanguageServer.Experimental.Protocol.Responses.GotoDefinition, ElixirLS.LanguageServer.Experimental.Protocol.Types.CallHierarchy.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.ChangeAnnotation, ElixirLS.LanguageServer.Experimental.Protocol.Types.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.CodeAction, ElixirLS.LanguageServer.Experimental.Protocol.Types.CodeAction.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.CodeAction.ClientCapabilities.CodeActionKind, ElixirLS.LanguageServer.Experimental.Protocol.Types.CodeAction.ClientCapabilities.CodeActionLiteralSupport, ElixirLS.LanguageServer.Experimental.Protocol.Types.CodeAction.ClientCapabilities.ResolveSupport, ElixirLS.LanguageServer.Experimental.Protocol.Types.CodeAction.Context, ElixirLS.LanguageServer.Experimental.Protocol.Types.CodeAction.Disabled, ElixirLS.LanguageServer.Experimental.Protocol.Types.CodeAction.Params, ElixirLS.LanguageServer.Experimental.Protocol.Types.CodeDescription, ElixirLS.LanguageServer.Experimental.Protocol.Types.CodeLens.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.CodeLens.Workspace.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.Command, ElixirLS.LanguageServer.Experimental.Protocol.Types.Completion.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.Completion.ClientCapabilities.CompletionItem, ElixirLS.LanguageServer.Experimental.Protocol.Types.Completion.ClientCapabilities.CompletionItemKind, ElixirLS.LanguageServer.Experimental.Protocol.Types.Completion.ClientCapabilities.CompletionList, ElixirLS.LanguageServer.Experimental.Protocol.Types.Completion.ClientCapabilities.InsertTextModeSupport, ElixirLS.LanguageServer.Experimental.Protocol.Types.Completion.ClientCapabilities.ResolveSupport, ElixirLS.LanguageServer.Experimental.Protocol.Types.Completion.ClientCapabilities.TagSupport, ElixirLS.LanguageServer.Experimental.Protocol.Types.CreateFile, ElixirLS.LanguageServer.Experimental.Protocol.Types.CreateFile.Options, ElixirLS.LanguageServer.Experimental.Protocol.Types.Declaration.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.Definition.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.DeleteFile, ElixirLS.LanguageServer.Experimental.Protocol.Types.DeleteFile.Options, ElixirLS.LanguageServer.Experimental.Protocol.Types.Diagnostic, ElixirLS.LanguageServer.Experimental.Protocol.Types.Diagnostic.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.Diagnostic.RelatedInformation, ElixirLS.LanguageServer.Experimental.Protocol.Types.Diagnostic.Workspace.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.DidChangeConfiguration.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.DidChangeWatchedFiles.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.Document.Color.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.Document.Formatting.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.Document.Formatting.Options, ElixirLS.LanguageServer.Experimental.Protocol.Types.Document.Formatting.Params, ElixirLS.LanguageServer.Experimental.Protocol.Types.Document.Highlight.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.Document.Link.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.Document.OnTypeFormatting.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.Document.RangeFormatting.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.Document.Symbol.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.Document.Symbol.ClientCapabilities.SymbolKind, ElixirLS.LanguageServer.Experimental.Protocol.Types.Document.Symbol.ClientCapabilities.TagSupport, ElixirLS.LanguageServer.Experimental.Protocol.Types.ExecuteCommand.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.FileOperation.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.FoldingRange.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.FoldingRange.ClientCapabilities.FoldingRange, ElixirLS.LanguageServer.Experimental.Protocol.Types.FoldingRange.ClientCapabilities.FoldingRangeKind, ElixirLS.LanguageServer.Experimental.Protocol.Types.Formatting.Options, ElixirLS.LanguageServer.Experimental.Protocol.Types.General.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.General.ClientCapabilities.StaleRequestSupport, ElixirLS.LanguageServer.Experimental.Protocol.Types.Hover.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.Implementation.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.Initialize.Params, ElixirLS.LanguageServer.Experimental.Protocol.Types.Initialize.Params.ClientInfo1, ElixirLS.LanguageServer.Experimental.Protocol.Types.InlayHint.ClientCapabilities, ElixirLS.LanguageServer.Experimental.Protocol.Types.InlayHint.ClientCapabilities.ResolveSuppor (truncated)
[Info  - 5:07:59 PM] Connection to server got closed. Server will restart.
true
Starting ElixirLS release v0.14.3
Running in /home/boschrexroth/wspc/elixir/rest_api
[Info  - 5:08:00 PM] Started ElixirLS v0.14.3
[Info  - 5:08:00 PM] Running in /home/boschrexroth/wspc/elixir/rest_api
[Info  - 5:08:00 PM] ElixirLS built with elixir "1.14.1" on OTP "24"
[Info  - 5:08:00 PM] Running on elixir "1.14.1 (compiled with Erlang/OTP 24)" on OTP "24"
[Info  - 5:08:00 PM] Elixir sources not found (checking in /home/build/elixir). Code navigation to Elixir modules disabled.
[Info  - 5:08:05 PM] Experimental server is disabled.
[Warn  - 5:08:06 PM] Did not receive workspace/didChangeConfiguration notification after 5 seconds. Using default settings.
[Info  - 5:08:07 PM] Loaded DETS databases in 36ms
[Info  - 5:08:07 PM] Starting build with MIX_ENV: test MIX_TARGET: host
[Info  - 5:08:07 PM] Compile took 411 milliseconds
[Info  - 5:08:07 PM] [ElixirLS WorkspaceSymbols] Indexing...
[Info  - 5:08:07 PM] [ElixirLS Dialyzer] Checking for stale beam files
[Info  - 5:08:07 PM] [ElixirLS Dialyzer] Found 0 changed files in 43 milliseconds
[Info  - 5:08:07 PM] [ElixirLS WorkspaceSymbols] Module discovery complete
[Info  - 5:08:07 PM] [ElixirLS Dialyzer] Analyzing 0 modules: []
[Info  - 5:08:07 PM] [ElixirLS Dialyzer] Analysis finished in 39 milliseconds
[Info  - 5:08:07 PM] Dialyzer analysis is up to date
[Info  - 5:08:07 PM] [ElixirLS WorkspaceSymbols] 30 callbacks added to index
[Info  - 5:08:08 PM] [ElixirLS Dialyzer] Writing manifest...
[Info  - 5:08:08 PM] [ElixirLS WorkspaceSymbols] 256 modules added to index
[Info  - 5:08:08 PM] [ElixirLS Dialyzer] Done writing manifest in 1078 milliseconds.
[Info  - 5:08:09 PM] [ElixirLS WorkspaceSymbols] 380 types added to index
[Info  - 5:08:14 PM] [ElixirLS WorkspaceSymbols] 5034 functions added to index
```

### Expected behavior

No Crash

**Comment by philippseith** [comment]
Why is this closed? It crashes...

**Comment by lukaszsamson** [comment]
Fix released in 0.14.5. If it is still affection you please post detailed repro steps or a project that reproduces it
